circle.yml

```diff
@@ -1,6 +1,6 @@
 machine:
   java:
-    version: oraclejdk7
+    version: oraclejdk8
```

Comment:
sorry, we need to build this with Java 7 because some consumers still use Java 7.